Source/Login/Backend/configuration/viper/identitites_configuration.go

```diff
@@ -1,9 +1,13 @@
 package viper
 
-import "github.com/spf13/viper"
+import (
+	"dolittle.io/login/identities/tenants"
+	"github.com/spf13/viper"
+)
 
 const (
 	identitiesCurrentUserCookieNameKey = "identities.cookie_name"
+	identitiesTenantNameMapKey         = "identities.tenants"
```

Comment:
I suggest that we change the name from "TenantNameMap" -> "TenantNames"
```suggestion
	identitiesTenantNamesKey         = "identities.tenants"
```